modules/gltf/gltf_document.cpp

```diff
@@ -3199,6 +3207,43 @@ Error GLTFDocument::_parse_images(Ref<GLTFState> state, const String &p_base_pat
 			state->images.push_back(Ref<Texture2D>());
 			continue;
 		}
+		if (state->discard_textures) {
+			state->images.push_back(Ref<Texture2D>());
+			continue;
+		}
+		if (state->extract_textures) {
```

Comment:
This will probably need changes to notify the importer that a texture was saved, but I guess its out of the scope of this PR.

Also, when just opening the scene (Before import options are passed) I think these flags need to be ignored since they are for previsualization.